server/lib/parties/party-service.ts

```diff
@@ -216,26 +220,33 @@ export default class PartyService {
     party.members.set(user.id, user)
     this.clientSocketsToPartyId.set(clientSockets, partyId)
 
-    this.publishToParty(partyId, {
+    this.publisher.publish(getPartyPath(party.id), {
       type: 'join',
       user,
     })
-
-    this.unsubscribeFromInvites(party, user)
     this.subscribeToParty(clientSockets, party)
   }
 
-  private unsubscribeFromInvites(party: PartyRecord, user: PartyUser) {
-    this.nydus.publish(getInvitesPath(party.id, user.id), { type: 'removeInvite' })
-    // TODO(2Pac): Remove the invite notification once the server-side notification system is in.
-    const userSockets = this.userSocketsManager.getById(user.id)
-    if (userSockets) {
-      userSockets.unsubscribe(getInvitesPath(party.id, user.id))
-    }
+  private async clearInviteNotification(partyId: string, user: PartyUser) {
+    this.notificationService
```

Comment:
This method is async, so it seems like the intention is that you can use `await`, but then you only attach promise handlers. If the intention is that we don't wait for this clearing to complete and it has no bearing on the initiating request, that's fine, but it probably shouldn't be an async function in that case